src/main/java/edu/ie3/datamodel/io/source/IdCoordinateSource.java

```diff
@@ -66,17 +66,15 @@
    *
    * @param coordinate the coordinate to look up the nearest neighbours for
    * @param n how many neighbours to look up
-   * @param allCoordinates the collection of points, ideally containing all available coordinates
+   * @param coordinates the collection of points
```

Comment:
did change the documentation because ideally you would only look through a minimal set of coordinates that include the nearest coordinates